syntax/go.vim

```diff
@@ -29,19 +29,19 @@ if exists('b:current_syntax')
 endif
 
 if !exists('g:go_highlight_array_whitespace_error')
-  let g:go_highlight_array_whitespace_error = 1
+  let g:go_highlight_array_whitespace_error = 0
```

Comment:
FWIW, I think the first three are valuable, but this does make things more consistent. I have a feeling most people are using vim-go for syntax anyway.